ubuntu-image

```diff
@@ -1,2 +1,28 @@
 #!/bin/sh
+
+if [ -n "$SNAP" ]; then
+    # If we're running as a snap, do some additional wrapping for classic
+    case $SNAP_ARCH in
+    amd64)
+        TRIPLET="x86_64-linux-gnu"
+        ;;
+    armhf)
+        TRIPLET="arm-linux-gnueabihf"
+        ;;
+    arm64)
+        TRIPLET="aarch64-linux-gnu"
+        ;;
+    *)
+        TRIPLET="$(uname -p)-linux-gnu"
+        ;;
+    esac
+
+    # We need to set the library paths for our snap to find the included libs.
+    export LD_LIBRARY_PATH=$SNAP/usr/lib:$SNAP/usr/lib/$TRIPLET:$SNAP/lib/$TRIPLET
+    export PYTHONPATH=$SNAP/lib/python3.5/site-packages:$PYTHONPATH
+    # Since we use some system-owned tools like mount, sudo or snap that we can't
+    # snappify, add some of the system paths to PATH
+    export PATH=$SNAP/usr/bin:$SNAP/bin:$SNAP/sbin:/usr/bin:/bin:$PATH
```

Comment:
Why do you include /usr/bin:/bin in the prepended path?  Should you not rely on snap already being present on $PATH?